tools/testbench/common_test.c

```diff
@@ -264,14 +264,14 @@ int tb_pipeline_params(struct testbench_prm *tp, struct ipc *ipc, struct pipelin
 /* print debug messages */
 void debug_print(char *message)
 {
-	if (debug)
+	if (host_trace_level >= LOG_LEVEL_DEBUG)
 		printf("debug: %s", message);
 }
 
 /* enable trace in testbench */
 void tb_enable_trace(bool enable)
 {
-	test_bench_trace = enable;
+	host_trace_level = enable;
```

Comment:
confused by this commit: (1) where `host_trace_level` is defined (apparently, it is defined somewhere, since this all works, or maybe it doesn't need to be, and (2) doesn't the above mean, that we set `host_trace_level` to either `LOG_LEVEL_ERROR` or to 0, which isn't even a valid error level?

Reply:
```debug``` colides easily and ```test_bench_trace``` tightly coupled any user code to testbench.

Reply:
@lgirdwood sorry, I mean it's a wrong assignment IMHO. Before `test_bench_trace` was effectively a boolean variable - it only took values of 0 and 1. Now `host_trace_level` is a printing verbosity **level** taking a value between `LOG_LEVEL_CRITICAL` (1) and `LOG_LEVEL_VERBOSE` (4). So this assignment should be fixed.

Reply:
@ranj063 can you update testbench to take a log level instead of a bool on the command line which can be passed above. Thanks!

Reply:
fixed now